Automattic/newspack-plugin
# fix: disable publicize for products

### All Submissions:






### Changes proposed in this Pull Request:



Disables Publicize for WooCommerce products. Most Newspack sites only use products for Newspack donations and self-serve listings, and these products should not be promoted publicly.

Closes #1507.

### How to test the changes in this Pull Request:

1. On your test site, connect Jetpack, go to **Jetpack > Settings > Sharing**, and enable Publicize. You don't have to connect any services to test this fix.

<img width="1008" alt="Screen Shot 2022-02-10 at 2 28 27 PM" src="https://user-images.githubusercontent.com/2230142/153499556-c69fdd3b-6a7d-4c5e-9f21-3a5ef2051a2d.png">

2. On `master`, go to **Products > Add New**. Observe a "Publicize" message in the Publish metabox. This indicates that Publicize is active for this post type, and if any social media services are connected via Jetpack, will automatically share the post to the connected services on publish.

<img width="284" alt="JVfOWm" src="https://user-images.githubusercontent.com/2230142/153499581-f489c601-082b-4a18-97c5-eff4820f5234.png">

3. Check out this branch, refresh the editor, and confirm that the Publicize message is no longer shown.

### Other information:
